app/stylesheet/miq-structured-list.scss

```diff
@@ -172,6 +172,30 @@
       margin: 10px;
     }
   }
+
+  &.tag_group {
+    .bx--structured-list-row {
+      display: flex;
+      flex-direction: column;
+
+      .label_header {
+        width: 100%;
+        min-width: 100%;
+        padding: 1rem;
+      }
+
+      .content_value {
+        padding-left: 1rem;
+
+        .multi_row_cell {
+          .sub_label {
+            width: 35%;
+            flex-grow: 1;
```

Comment:
The flex-grow sometimes causes a bit of overlap. Maybe add a minimum distance between the columns?
![image](https://github.com/ManageIQ/manageiq-ui-classic/assets/64800041/57772990-c3ef-4042-b7df-8f00a56fad54)

Reply:
Thanks @DavidResende0 , will fix this.

Reply:
Corrected the alignment issues
<img width="548" alt="image" src="https://github.com/ManageIQ/manageiq-ui-classic/assets/87487049/ff0f465c-2b2f-4df5-b878-1151cd5b1880">
<img width="545" alt="image" src="https://github.com/ManageIQ/manageiq-ui-classic/assets/87487049/4c0f9649-7a9a-4e6d-b5c4-44b2cb64d066">